src/test/java/vehicle/PutVehicleTechnicalRecords.java

```diff
@@ -2113,7 +2113,8 @@ public void testPostVehicleTechRecordTrlPrsNotifiableAlteration(){
         vehicleTechnicalRecordsSteps.valueForFieldInPathShouldContains("[0].techRecord.find { it.statusCode == 'current' }.createdAt", new SimpleDateFormat("yyyy-MM-dd").format(date));
     }
 
-    @WithTag("Vtm")
+//    @WithTag("Vtm")
```

Comment:
Same here, is commented out code required?

Reply:
Removed

Reply:
Restored